docs/getting-started/README.md

```diff
@@ -0,0 +1,74 @@
+# Getting started
+
+## System requirements
+
+| Software | Minimum Version |
+| ------------- | ------------- |
+| [Node.js](https://nodejs.org/en/) | 12.x+ |
+| [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)  | 3.6+ |
+| [ImageMagick](https://imagemagick.org/script/download.php#macosx) (optional) | Any |
+
+See the links above to install each. To check that these are installed in your environment, try the following commands:
+
+```bash
+node -v && npm -v
+# This will display your Node.js version and npm version (installed with Node),
+# if installed.
+mongod --version
+# This will display your MongoDB version, if installed.
+which convert && which identify
+# This will display the location of the ImageMagick utilities, if installed.
+```
+
+::: tip NOTE
+ImageMagick is optional, but recommended. It provides the `convert` and `identify` command line tools, which Apostrophe uses to scale and crop images quickly. If you do not install it Apostrophe can still handle image uploads, though more slowly, using [Jimp](https://www.npmjs.com/package/jimp).
+:::
+
+<!-- ## TODO: The Apostrophe CLI tool -->
+
+## Setting up your Apostrophe project
+
+<!-- TODO: Update with CLI info when ready. -->
+The easiest way to get started with Apostrophe is to use the starter boilerplate project. On the command line, clone the boilerplate:
+
+```bash
+git clone https://github.com/apostrophecms/a3-boilerplate apos-app
+```
+
+If you want to change the project directory name, please do so. We will continue referring to `apos-app`.
+
+::: tip NOTE
+There is a CLI tool for Apostrophe 2 with commands to create projects, among other tasks. That will support Apostrophe 3 closer to the 3.0 stable release.
+:::
+
+Open the project directory in your code editor. We'll want to change one line before starting up so the database is created correctly. In the `app.js` file, find the `shortname` setting and change it to match your project (only letters, digits, hyphens and/or underscores). By default, this will be the name of your database:
+
+```javascript
+// app.js
+const path = require('path');
+
+require('apostrophe')({
+  shortName: 'apos-app', // 👈
+  modules: {
+  // ...
+```
+
+Excellent. Back in your terminal we'll install dependencies:
+
+```bash
+npm install
+# Or `yarn install`, if you prefer. We'll stick to npm commands.
+```
+
+You could start the site up now, but you'll need to create the first user if you want to log in. After running the command, Apostrophe will ask you to enter a password for this user.
```

Comment:
```suggestion
Before startup, you'll need to create a user so that you can log in to the editor. After running the command, Apostrophe will ask you to enter a password for this user.
```